static/app/utils/replays/getFrameDetails.tsx

```diff
@@ -274,24 +276,34 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
     title: 'Navigation',
     icon: <IconLocation size="xs" />,
   }),
-  'largest-contentful-paint': (frame: WebVitalFrame) => ({
-    color: 'gray300',
-    description:
-      typeof frame.data.value === 'number' ? (
-        `${Math.round(frame.data.value)}ms`
-      ) : (
-        <Tooltip
-          title={t(
-            'This replay uses a SDK version that is subject to inaccurate LCP values. Please upgrade to the latest version for best results if you have not already done so.'
-          )}
-        >
-          <IconWarning />
-        </Tooltip>
-      ),
-    tabKey: TabKey.NETWORK,
-    title: 'LCP',
-    icon: <IconInfo size="xs" />,
-  }),
+  'web-vital': (frame: WebVitalFrame) => {
+    switch (frame.data.rating) {
+      case 'good':
+        return {
+          color: 'green300',
+          description: `Good ${Math.round(frame.data.value * 100) / 100}ms`,
+          tabKey: TabKey.NETWORK,
+          title: frame.description.replaceAll('-', ' '),
```

Comment:
Might be able to call `toTitleCase(explodeSlug(frame.description))` for these titles